### PR TITLE
Benchmark: Add all core dependencies

### PR DESCRIPTION
**Pull request by jonludlam**
This commit adjusts the set of libraries used as a benchmark. It adds approximately the minimal set of libraries to ensure a self-consistent set of odocs without unresolved module dependencies.

I found this particularly useful when testing some recent changes - changes that showed no difference in output when run on the 'normal' set of packages, but a large number of differences when run on the extended list.

### CODE REVIEW

File: doc/library_mlds/base_bigstring.mld [hunk 1]
```diff
@@ -0,0 +1,2 @@
+{0 Base_bigstring}
```
Comment (Julow) [inline doc/library_mlds/base_bigstring.mld:1]: These pages are too hard to create and delete. The driver should generate them instead.

File: doc/library_mlds/ppx_expect.make_corrected_file.mld [hunk 2]
```diff
@@ -0,0 +1 @@
+{0 ppx_expect.make_corrected_file}
```
Comment (panglesd) [inline doc/library_mlds/ppx_expect.make_corrected_file.mld:1]: Is this file added on purpose?

Comment (jonludlam) [reply doc/library_mlds/ppx_expect.make_corrected_file.mld:1]: Nope, thanks!

Comment (jonludlam) [reply doc/library_mlds/ppx_expect.make_corrected_file.mld]: Fixed.

**Review by Julow — APPROVED** [review]
More unnecessary pages can be removed: `base`, `base_quickcheck`, `core`, `ppx_hash`, etc.. as they are not expected in a build without `extra_deps`.

Otherwise, looks good.

**Pull request merged**